src/Conversations/ConversationFilters.php

```diff
@@ -74,16 +82,50 @@ public function getParams(): array
             'query' => $this->query,
         ];
 
-        if (is_array($this->tag)) {
+        if (\is_array($this->tag)) {
             $params['tag'] = implode(',', $this->tag);
         }
 
+        if (\is_array($this->customFieldIds)) {
+            $params['customFieldIds'] = implode(',', $this->customFieldIds);
+        }
+
         // Filter out null values
         return array_filter($params, function ($param) {
             return $param !== null;
         });
     }
 
+    /**
+     * @param int   $id
+     * @param mixed $value
+     *
+     * @return ConversationFilters
+     */
+    public function withCustomField(int $id, $value)
+    {
+        $filters = clone $this;
+        if ($this->customFieldIds === null) {
+            $this->customFieldIds = [];
+        }
+        $filters->customFieldIds[] = "$id:$value";
+
+        return $filters;
+    }
+
+    /**
+     * @param array $fields
+     *
+     * @return ConversationFilters
+     */
+    public function withCustomFields(array $fields)
```

Comment:
Can this be type hinted return here and above `: ConversationFilters`?

Reply:
doh! I'll fix that up....